types/babel-plugin-macros/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for babel-plugin-macros 2.6
+// Project: https://github.com/kentcdodds/babel-plugin-macros
+// Definitions by: Billy Kwok <https://github.com/billykwok>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
This is the _minimum_ TypeScript version required. You should set it to the lowest value possible.

Reply:
Thanks for your comment. Fixed.